src/app/core/actions/transform/TranslateAction.ts

```diff
@@ -1,40 +1,66 @@
 import {Vector} from "Vector";
-
 import {Action} from "core/actions/Action";
-
 import {Component} from "core/models/Component";
-
 import {SnapPos} from "./SnapUtils";
 
-// Translate can be applied to single components,
```

Comment:
Not sure if there is really any need to change the spacing between imports that already exists. That said, there should be two empty lines between the last import and the first line of whatever else (in this case a comment)